### ISSUE TITLE
Duda sobre uso de archivo [NombresComunes.txt]

### ISSUE THREAD

**Issue by EdgardoRios** [sbosio/rla-es#142]
Hola, espero estén muy bien:

El día de hoy, después de agregar la palabra **jurista** en el archivo [NombresMasculinosFemeninos.txt], me dí cuenta que ya existía en el archivo [NombresComunes.txt].

¿Me pudieran por favor explicar el criterio que debería usar para el archivo [NombresComunes.txt]?

Saludos.
Edgardo

**Comment by cosmoscalibur** [comment]
Inquietud que también manifesté en #138 .

**Comment by sbosio** [comment]
Las categorías gramaticales las saqué originalmente del sitio del DRAE
cuando estaba publicada la revisión 22. Ahora parece que la han modificado.

El archivo "NombresMasculinosFemeninos.txt" supuestamente debería contener
únicamente los nombres que admiten masculino y femenino, pero cuya grafía
cambia entre uno y otro, p. ej.: "abogado/abogada" y generalmente deberían
todos estar escritos en masculino singular, con la bandera "G" que genera
el femenino y la bandera "S" para el plural del masculino (el plural del
femenino lo genera la misma regla G).

El archivo "NombresComunes.txt" recogía todos los lemas de la revisión 22
que estaban clasificados como "nombres comunes en cuanto al género", es
decir los nombres donde la grafía no cambia entre el masculino y el
femenino. Es el caso del lema que quisiste agregar. Es correcto tanto "el
jurista" como "la jurista", pero en cada caso el contexto indica si
representa al masculino o al femenino (a un hombre o una mujer). No debería
aparecer ninguno con la bandera "G", aunque si revisás el archivo los hay y
son errores que deberían moverse al archivo correcto.

El archivo "NombresAmbiguos.txt" contenía los lemas que estaban
clasificados como "nombres ambiguos en cuanto al género", p. ej.: "sartén",
pero ahora se los ha clasificado distinto, y ese lema aparece como nombre
femenino, pero con la leyenda que en américa y españa se usa también como
masculino. Es decir que es correcto decir "el sartén" o "la sartén" (quizás
con preferencia sobre este último), pero no porque sean dos objetos
distintos, uno identificado con el masculino y otro con el femenino, sino
porque no existe una asociación clara con un género o el otro. Acá tampoco
deberían aparecer lemas con la bandera "G", pero también hay varios si te
ponés a revisar.

Saludos,


2017-09-07 13:25 GMT-03:00 Edward Yesid Villegas Pulgarin <
notifications@github.com>:

**Comment by Almorca** [comment]
Puestos que todos los ficheros guardan relación con la gramática y los ficheros NombresComunes y NombresAmbiguos ya no, ¿no sería mejor unificarlos con el fichero NombresMasculinos y así evitar dudas?

**Comment by cosmoscalibur** [comment]
@Almorca , aunque con esto el fichero `NombresComunes.txt` si es necesario. Si bien no aparece en las definiciones con una mención directa a este nombre, puede ser útil diferenciar los lemas que no poseen cambio de grafía según el género (sería importante si se considera la herramienta de corrección de gramática).  
Pero en caso de considerarlo, los casos ambiguos y comunes no pertenecerían a `NombresMasculinos.txt` sino a `NombresMasculinosFemeninos.txt`.

**Comment by sbosio** [comment]
@cosmoscalibur: para mantener el criterio correcto, en caso de eliminar cualquiera de esos ficheros (NombresComunes.txt y NombresAmbiguos.txt) cada uno de los lemas incluidos debería agregarse a los ficheros de NombresMasculinos.txt y NombresFemeninos.txt por separado.
El fichero de NombresMasculinosFemeninos.txt debería contener únicamente sustantivos masculinos singulares y usar obligatoriamente las banderas "G" y "S". Remarco el "debería", porque actualmente no es así y hay varios errores que depurar y enviar a su archivo correspondiente.

**Comment by cosmoscalibur** [comment]
¿Si amerita duplicar lemas cuando se puede tener dicho archivo? En mi caso, opino que tal vez el archivo `NombresAmbiguos.txt` se una con `NombresComunes.txt`, pues cumplen para fines prácticos la misma función (aunque hay que revisar los lemas de `NombresAmbiguos.txt` para los casos que en la nueva recomendación de la RAE ya sea un solo género como aclaraste en otra discusión).  
De hecho, si se aspira a trabajar en un futuro a generar el insumo de LT, también existe la categoría de nombre común en las [reglas que usan para el español](https://github.com/languagetool-org/languagetool/blob/master/languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/etiquetas-eagles.md#4-nombres).

**Comment by cosmoscalibur** [comment]
Tal vez la confusión que pueda provocarse (como lo referencia @Almorca ), podría solventarse con una página en la wiki que aclare el uso adecuado de los archivos. Yo puedo dedicarle ratos durante la semana para redactar la página.

**Comment by sbosio** [comment]
No, yo creo que no amerita la duplicación. Creo que como está es correcto.
No alcanzo a saber si tiene alguna utilidad mantener los dos ficheros o
integrar en uno único. Conceptualmente hay una diferencia, pero a los usos
gramaticales no se me ocurre algún ejemplo donde sea importante conocer a
cuál caso corresponde un lema.

**Comment by cosmoscalibur** [comment]
Creo que esto da mayor claridad al tema y ya veo una utilidad desde la gramática. Creo que superflua para uno como humano, pero para una herramienta de revisión de gramática sería de utilidad -en LT permitiría distinguir la clasificación semántica cuando sea implementada-. De [wikilengua](http://www.wikilengua.org/index.php/G%C3%A9nero_gramatical), asociado a Fundéu:  

>La denominación de palabra común (en cuanto al género) se aplica a las que aluden a personas y que tienen una única forma para ambos géneros (el atleta, la atleta), mientras que la de palabra ambigua (en cuanto al género) alude a las de cosas y conceptos que se pueden usar en ambos géneros (el mar, la mar).

**Comment by cosmoscalibur** [comment]
Como mencione antes respecto a redactar un documento con aclaraciones del uso de archivos, me encuentro terminando dicha función. Pero tengo una inquietud:  
El archivo `Frases.txt` contiene lemas que corresponden a locuciones verbales pero ya existe un archivo `Locuciones.txt`. ¿Debemos migrar los lemas o cual es la razón de dicho archivo?  
El estado actual de la redacción se encuentra en la wiki de mi derivado [Clasificación de lemas (uso de archivos)](https://github.com/cosmoscalibur/rla-es/wiki/Clasificaci%C3%B3n-de-lemas-(uso-de-archivos)) en espera de comentarios para ubicarlo aquí.

**Comment by RickieES** [comment]
La razón de que existan ambos archivos no la sé, pero me parece razonable migrar Frases.txt a Locuciones.txt si no hay un motivo real para la separación.

Excelente el trabajo que has hecho en la página de clasificación de lemas. 👍
